Add taking the address of math constants to known issues (#461)
kokkos/kokkos-core-wiki: docs/source/API/core/numerics/mathematical-constants.rst

Notes
-----

.. _KnownIssues: ../../../known-issues.html#mathematical-constants

.. |KnownIssues| replace:: known issues

* The mathematical constants are available in ``Kokkos::Experimental::`` since Kokkos 3.6
* They were "promoted" to the ``Kokkos::numbers`` namespace in 4.0

> * Avoid taking the address of mathematical constants in device code.  It is not supported by some toolchains, hence not portable.  (See |KnownIssues|_)

Reviewer: ```suggestion
* Passing mathematical constants by reference or taking their address in device code is not supported by some toolchains and hence not portable.  (See |KnownIssues|_)
```
